tests/pygeoapi-test-config-postgres-manager.yml

```diff
@@ -0,0 +1,111 @@
+# =================================================================
+#
+# Authors: Francesco Martinelli <francesco.martinelli@ingv.it>
+#
+# Copyright (c) 2024 Francesco Martinelli
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+server:
+    bind:
+        host: 0.0.0.0
+        port: 5000
+    url: http://localhost:5000/
+    mimetype: application/json; charset=UTF-8
+    encoding: utf-8
+    gzip: false
+    languages:
+        # First language is the default language
+        - en-US
+        - fr-CA
+    cors: true
+    pretty_print: true
+    limit: 10
+    # templates: /path/to/templates
+    map:
+        url: https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png
+        attribution: '<a href="https://wikimediafoundation.org/wiki/Maps_Terms_of_Use">Wikimedia maps</a> | Map data &copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap contributors</a>'
+    manager:
+        name: 'pygeoapi.process.manager.postgresdb_.PostgreSqlDBManager'
```

Comment:
To keep consistent with the definition of `manager.connection`, suggest to define as a DB string formatting as per RFC1738.  Example:`postgresql://postgres:postgres@localhost:5432/ogc_api`

Reply:
I followed the configuration present in the test for PostgreSQL providers: `test_postgresql_provider.py`.
I find it more clear the key-value configuration rather then an URI.

The used library (`psycopg2`), on receiving an URI to make a connection, delegate the parsing to `libqp` (https://www.psycopg.org/docs/extensions.html#psycopg2.extensions.parse_dsn). Then psycopg2 depends also on `libqp` version (9.2 is required to parse URI).

The documentation of PostgreSQL reports that URI connection strings (https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) are accepted when following [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) (an update of RFC 1738).

I will modify the code to accept **also** URI,
and the test configuration file `pygeoapi-test-config-postgres-manager.yml` to insert comment on the possibility to use URI as well.